test/run_tests.sh

```diff
@@ -7,6 +7,9 @@ current_dir=$(dirname "$(readlink -f "$0")")
 export KB_DEPLOYMENT_CONFIG="$current_dir"/deploy.cfg
 export PYTHONPATH="$current_dir"/../lib:"$PYTHONPATH"
 
+# ensure there's a valid scratch dir and an assets dir
+mkdir -p "$current_dir"/../scratch/nginx
```

Comment:
Should this get cleaned up at the end of a test run? I don't **think** it matters, but it might affect tests that expect an empty directory.

Reply:
Good point -- it might be an idea to move the configuration of that dir into the tests and make it a temp dir. I'll do that in an upcoming PR.